modules/proto/src/smithytranslate/proto3/internals/Compiler.scala

```diff
@@ -231,7 +243,7 @@ private[proto3] class Compiler(model: Model, allShapes: Boolean) {
           .getTrait(classOf[ProtoNumTypeTrait])
           .toScala
           .map(_.getNumType())
-        val maybeType = shape.accept(typeVisitor(false, maybeNumType))
+        val maybeType = shape.accept(typeVisitor(false, false, maybeNumType))
```

Comment:
It'd be helpful here to use named arguments for the booleans so we can see what's false without jumping to def.